src/main/resources/openapiv2.json

```diff
@@ -0,0 +1,876 @@
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "ID of the consumer group to which the consumer belongs.",
+                    "required": true,
+                    "type": "string"
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "Name of the consumer.",
+                    "required": true,
+                    "type": "string"
+                }
+            ]
+        },
+        "/consumers/{groupid}/instances/{name}/assignments": {
+            "post": {
+                "tags": [
+                    "Consumers"
+                ],
+                "description": "Assigns one or more topic partitions to a consumer.",
+                "operationId": "assign",
+                "parameters": [
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "description": "List of topic partitions to assign to the consumer.",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/PartitionsForPositions"
+                        }
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "OK"
+                    },
+                    "404": {
+                        "description": "The specified consumer instance was not found."
+                    },
+                    "409": {
+                        "description": "Subscription to topics, partitions, and pattern are mutually exclusive."
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "ID of the consumer group to which the consumer belongs.",
+                    "required": true,
+                    "type": "string"
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "Name of the consumer to which you want to assign topic partitions.",
+                    "required": true,
+                    "type": "string"
+                }
+            ]
+        }
+    },
+    "definitions": {
+        "ApiResponse": {
+            "type": "object",
+            "properties": {
+                "code": {
+                    "format": "int32",
+                    "type": "integer"
+                },
+                "type": {
+                    "type": "string"
+                },
+                "message": {
+                    "type": "string"
+                }
+            }
+        },
+        "Consumer": {
+            "title": "Root Type for Consumer",
+            "description": "The root of the Consumer type's schema.",
```

Comment:
Not sure what this means. Is it 'The schema for the consumer root type'? 
This comment applies to all descriptions that begin: 'The root of the...'

Reply:
Good point. @stanlyDoge - Should the "Models" section even be visible in the API documentation?

Reply:
These titles are autogenerated. I can change them, but I think it is necessary to have models in this spec file.